components/src/dynamo/planner/core/base.py

```
@@ -89,125 +64,99 @@ def _engine_caps(
         self.prometheus_metrics = PlannerPrometheusMetrics()
         if self.prometheus_port != 0:
             try:
                 start_http_server(self.prometheus_port)
                 logger.info(
-                    f"Started Prometheus metrics server on port {self.prometheus_port}"
+                    "Started Prometheus metrics server on port %s",
+                    self.prometheus_port,
                 )
-            except Exception as e:
-                logger.error(f"Failed to start Prometheus metrics server: {e}")
+            except Exception as exc:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)